docusaurus/docs/creating-a-plugin.md

```diff
@@ -0,0 +1,67 @@
+---
+id: creating-a-plugin
+title: Creating a Plugin
+---
+
+To create a new Grafana plugin run the following command in your shell:
+
+```shell
+npx @grafana/create-plugin
+```
+
+:::info
+The answers to the name, organization and type of plugin prompts will be combined to create the plugin directory name and the plugin id.
+
+```
+? What is going to be the name of your plugin? mongodb
+? What is the organization name of your plugin? grafana
+? What type of plugin would you like? datasource
+```
+
+In the above example this will result in the directory and plugin id being named `grafana-mongodb-datasource`
```

Comment:
I suggest changing "In the above example this will result in the directory and plugin id being named `grafana-mongodb-datasource`" to "In the above example, this will cause the directory and plugin ID to be named `grafana-mongodb-datasource`